pkg/console/metrics/metrics.go

```diff
@@ -30,3 +35,53 @@ func Register() *ConsoleMetrics {
 	})
 	return singleton
 }
+
+func HandleConsoleURL(oldURL, newURL string) {
+	// don't call before register
+	if !singleton.initialized {
```

Comment:
Can this ever happen?

Reply:
Not at present, no.  
Thinking about future additions and establishing a simple pattern.

Reply:
I'd remove this check if it's not possible now and add it when we have a specific need.